packages/x-charts/src/SparkLineChart/SparkLineChart.tsx

```diff
@@ -101,6 +101,12 @@ export interface SparkLineChartProps
    * }
    */
   margin?: Partial<CardinalDirections<number>>;
+  /**
+   * If `true`, bar animations are skiped.
+   * @deprecated In v7 animation are skiped for sparkline.
```

Comment:
```suggestion
   * @deprecated In v7 animations are skipped by default.
```

Reply:
> by default.

It's not exactly by default, they are just removed, assuming nobody needs it. We could add a props `enableAnimation` in v7 but for now it does not exist

Reply:
Gotcha. Updated the suggestion. 😉

Reply:
It was mostly to make sure you are not choked that v7 removes this feature ;)